h/static/scripts/test/annotation-ui-controller-test.js

```diff
@@ -22,11 +24,15 @@ describe('AnnotationUIController', function () {
     $scope = $rootScope.$new();
     $scope.search = {};
 
-    annotationUI = {
-      tool: 'comment',
+    var store = createFakeStore({
```

Comment:
Soooo... I'm looking at `annotation-ui.js`, and I'm having difficulty seeing why you couldn't use a real `annotationUI` object? What am I missing?